sdk/include/reactos/shlwapi_undoc.h

```diff
@@ -324,6 +325,12 @@ IContextMenu_Invoke(
 
 DWORD WINAPI SHGetObjectCompatFlags(IUnknown *pUnk, const CLSID *clsid);
 
+BOOL WINAPI
```

Comment:
I would suggest you move this function definition just after PathFileExistsDefExtW above (line 311/312)

Reply:
Please consider this too :point_up:

Reply:
Fixed.